.github/workflows/lmi-dist-deps-build.yml

```diff
@@ -91,8 +91,8 @@ jobs:
             name: build-artifacts
       - name: upload to S3
         run: |
-          aws s3 cp flash_attn-2*.whl s3://djl-ai-staging/publish/flash_attn/cu121-pt212/
-          aws s3 cp vllm*.whl s3://djl-ai-staging/publish/vllm/cu121-pt212/
+          aws s3 cp flash_attn-2*.whl s3://djl-ai-staging/publish/flash_attn/cu121-pt230/
```

Comment:
this should be cu124 right? Why we stick with cu121?

Reply:
We are building against `torch==2.3.0+cu121`. I guess the question is which cuda version we want to use in the path, the torch one or of the base nvidia container we used